src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -21,10 +23,33 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcore50aot_Release|AnyCPU'" />
-  <ItemGroup Condition="'$(TargetGroup)'!='net46'">
-    <Compile Include=".\**\*.cs" />
+  <ItemGroup Condition="'$(TargetGroup)'!='net46' And '$(TargetGroup)'!='net463'">
+    <Compile Include="System\Runtime\Serialization\CollectionDataContractAttribute.cs" />
+    <Compile Include="System\Runtime\Serialization\ContractNamespaceAttribute.cs" />
+    <Compile Include="System\Runtime\Serialization\DataContractAttribute.cs" />
+    <Compile Include="System\Runtime\Serialization\DataMemberAttribute.cs" />
+    <Compile Include="System\Runtime\Serialization\EnumMemberAttribute.cs" />
+    <Compile Include="System\Runtime\Serialization\IgnoreDataMemberAttribute.cs" />
+    <Compile Include="System\Runtime\Serialization\InvalidDataContractException.cs" />
+    <Compile Include="System\Runtime\Serialization\ISerializationSurrogateProvider.cs" />
+    <Compile Include="System\Runtime\Serialization\KnownTypeAttribute.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.3' Or '$(TargetGroup)'=='netcore50aot'">
+    <Compile Include="$(CommonPath)\System\Runtime\Serialization\OnDeserializedAttribute.cs" />
```

Comment:
yes, this is exactly the case where things would be easier with Eric's changes. I'll make sure that whoever onboards those changes(I suppose that would be me :smile:) does it for this as well.